mautrix_telegram/__main__.py

```diff
@@ -149,6 +155,8 @@ async def resend_bridge_info(self) -> None:
     def prepare_stop(self) -> None:
         if self.periodic_sync_task:
             self.periodic_sync_task.cancel()
+        if self.as_bridge_liveness_task:
```

Comment:
This will raise `AttributeError` if it's not defined, see comment about defining it as `None` for the instance.

Reply:
Set to None in d4b5e0b.